onnxscript/ir/_io.py

```diff
@@ -35,16 +35,75 @@ def load(path: str | os.PathLike, format: str | None = None) -> _core.Model:
     return model
 
 
-def save(model: _core.Model, path: str | os.PathLike, format: str | None = None) -> None:
+def save(
+    model: _core.Model,
+    path: str | os.PathLike,
+    format: str | None = None,
+    external_data: str | os.PathLike | None = None,
```

Comment:
nit suggestion: maybe name it as external_data_path to clarify it is a path to file?

Reply:
Since the option is intended to provided by keyword, I find `external_data` to be more concise and as clear. Let me know if you think differently?

Reply:
The parameter does not exist in ONNX standard API. It is controlled by a boolean flag and a location parameter. I consider the whole combination a little confusing (`location` itself is a strange naming for the api) so I deviated from it entirely.